serverlessworkflow/modules/ROOT/pages/release_notes.adoc

```diff
@@ -0,0 +1,21 @@
+= What's new on {page-component-display-version}
+:compat-mode!:
+
+== Notable Changes
+
+. link:https://issues.redhat.com/browse/KOGITO-7620[KOGITO-7620] - Improve events consumption thread pools configs
```

Comment:
I'd suggest using bullet points instead of numbers. We use numbers when describing steps. 
```suggestion
* link:https://issues.redhat.com/browse/KOGITO-7620[KOGITO-7620] - Improve events consumption thread pools configs
```